src/main/java/org/opensearch/performanceanalyzer/reader/MetricPropertiesConfig.java

```diff
@@ -305,12 +306,13 @@ private MetricPropertiesConfig() {
                         AllMetrics.ShardIndexingPressureValue.values(),
                         createFileHandler(metricPathMap.get(MetricName.SHARD_INDEXING_PRESSURE))));
         metricName2Property.put(
-                MetricName.MASTER_CLUSTER_UPDATE_STATS,
+                MetricName.CLUSTER_MANAGER_CLUSTER_UPDATE_STATS,
                 new MetricProperties(
                         MetricProperties.EMPTY_DIMENSION,
-                        AllMetrics.MasterClusterUpdateStatsValue.values(),
+                        AllMetrics.ClusterManagerClusterUpdateStatsValue.values(),
                         createFileHandler(
-                                metricPathMap.get(MetricName.MASTER_CLUSTER_UPDATE_STATS))));
+                                metricPathMap.get(
+                                        MetricName.CLUSTER_MANAGER_CLUSTER_UPDATE_STATS))));
```

Comment:
nit : can be a single line
```suggestion
                                metricPathMap.get(MetricName.CLUSTER_MANAGER_CLUSTER_UPDATE_STATS))));
```